MAINTAINERS.md

```diff
@@ -0,0 +1,13 @@
+# Maintainers 👔
+
+Tips & tricks for maintainers of Tickety-Tick.
+
+## Sign add-on for Firefox
+
+Offering a (beta-) version for download requires self-signing the add-on through [addons.mozilla.org](https://extensionworkshop.com/documentation/publish/#distribute-your-signed-extension). This generates a `.xpi` for installing the extension in Firefox.
+
+Find the API credentials in 1password and run:
+
+```
+yarn build:firefox && yarn sign 
```

Comment:
Will this ask for the credentials or how does it work?

Reply:
I'd provide them via environment variables. The command in 1Password looks like this:
```
WEB_EXT_API_KEY='xxx' WEB_EXT_API_SECRET='xxx' yarn build:firefox && yarn sign
```

I'll update the example to ☝️ for clarity.